third_party/mpi/BUILD

```diff
@@ -19,4 +19,5 @@ cc_library(
         "//conditions:default": ["libmpi.so"],
     }),
     hdrs = ["mpi.h", "mpi_portable_platform.h"],
```

Comment:
There is no guarentee that this header will be available in the future, so it represents a portability problem on multiple levels.  I don't see why you need this header at all, but if you like it, include it TF.  The license is permissive and it is standalone (it is originally from GASNet, not Open-MPI).